cmd/gitops-server/cmd/cmd.go

```diff
@@ -127,13 +130,26 @@ func runCmd(cmd *cobra.Command, args []string) error {
 
 	oidcConfig := options.OIDC
 
+	if options.OIDCSecret != auth.DefaultOIDCAuthSecretName {
+		log.V(logger.LogLevelDebug).Info("Reading OIDC configuration from alternate secret", "secretName", options.OIDCSecret)
+	}
+
 	// If OIDC auth secret is found prefer that over CLI parameters
 	var secret corev1.Secret
 	if err := rawClient.Get(cmd.Context(), client.ObjectKey{
 		Namespace: v1alpha1.DefaultNamespace,
-		Name:      auth.OIDCAuthSecretName,
+		Name:      options.OIDCSecret,
 	}, &secret); err == nil {
+		if len(options.OIDC.ClientSecret) > 0 && len(secret.Data["clientSecret"]) > 0 {
+			log.V(logger.LogLevelWarn).Info("OIDC client configured by both CLI and secret. CLI values will be overridden.")
+		}
 		oidcConfig = auth.NewOIDCConfigFromSecret(secret)
+	} else if err != nil {
+		log.V(logger.LogLevelDebug).Info("Could not read OIDC secret", "secretName", options.OIDCSecret, "error", err)
```

Comment:
Maybe use `logger.LogLevelError` ?

Reply:
>We should also try and standardise on the log string format.

Sure. Out of scope for this PR.

Reply:
>Maybe use `logger.LogLevelError` ?

It's not an error that you haven't set up OIDC login.

Reply:
> > Maybe use `logger.LogLevelError` ?
> 
> It's not an error that you haven't set up OIDC login.

This depends on the nature of the error, yes, if the secret is not found, it's "Info", but if it's not, it's perhaps Error?